packages/studio/src/LinkWidget.jsx

```diff
@@ -35,10 +37,17 @@ const AdditionalInfo = styled.div`
 const TEST_URL =
   "https%3A%2F%2Fraw.githubusercontent.com%2Fsgenoud%2Freplicad%2Fmain%2Fpackages%2Freplicad-docs%2Fexamples%2FsimpleVase.js";
 
+const loadCode = async (rawCode) => {
```

Comment:
Why not have it in utils and imported in both places where you need it (it feels like a nightmare to debug if we do a modification in one and expect it to work in the other...)

Reply:
For sure! I wasn't familiar with the project history, and whether it was OK to tie the two areas together like this. I'll move the code over :+1:

Reply:
As we are probably going to add a more consequent share modal anyway where the code will be linked.